README.md

```diff
@@ -7,8 +7,8 @@
 | № | Имя   | Любимый фильм                          |
 | - |:-----:| --------------------------------------:|
 | 0 | Ава   | Три идиота                             |
-| 1 | Павел | Криминальное чтиво                     |
-| 2 |  Камила      |       Кролик ДжоДжо             |
-| 3 | Андрей| Храброе Сердце                         |
- 
-
+| 1 | Катя  | Такси                                  |                          
+| 2 | Павел | Криминальное чтиво                     |                                 
+| 3 | Павел | Криминальное чтиво                     |
```

Comment:
why pavel is here twice?

Reply:
I guess he added himself there twice

Reply:
the green text is your changes. Please, be accurate and resolve.